config/ib_api.yaml

```diff
@@ -174,6 +184,7 @@ paths:
           name: composeId
           schema:
             type: string
+            format: uuid
             example: '123e4567-e89b-12d3-a456-426655440000'
```

Comment:
These all changes are just format hint, in Go this maps to Google's UUID so I think this should be safe to merge and it appears to be a compatible change.